.buildkite/scripts/verify_promotion.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# After uploading a container and then promoting it, we must verify
+# that the move actually happened as we expected.
+
+set -euo pipefail
+
+# Get access to our cloudsmith "binary"
+# shellcheck source-path=SCRIPTDIR
+source "$(dirname "${BASH_SOURCE[0]}")/../../lib/cloudsmith.sh"
+# shellcheck source-path=SCRIPTDIR
+source "$(dirname "${BASH_SOURCE[0]}")/../../lib/ops.sh"
+# shellcheck source-path=SCRIPTDIR
+source "$(dirname "${BASH_SOURCE[0]}")/../../lib/log.sh"
+
+readonly image_name="cloudsmith-buildkite-plugin-verify-test"
+readonly image_tag="${BUILDKITE_BUILD_ID}"
+readonly fully_qualified_image_name="${image_name}:${image_tag}"
+
+query() {
+    local -r _repo="${1}"
+
+    # In this case, we don't care about the error message that
+    # `get_unique_identifier` can return.
+    get_unique_identifier grapl "${_repo}" "name:${image_name} version:${image_tag}" 2> /dev/null
+}
+
+echo "--- :cloudsmith: Verifying promotion out of source repository"
+if package_id=$(query testing-stage1); then
```

Comment:
would be nice to document somewhere that `testing-stage1/2` are meant specifically for testing *this* plugin